script/QA/QA_08.txt

```diff
@@ -125,7 +125,7 @@ until you are able to team up with someone you can rely on.
 [sha:bf9416a8c1d6958167f9a6d3f14295b9bf9c79f5]{
 -- Page 169, Offset 43072.
 -- まさに<ＳＰＹ|エスピーワイ>！
-Just like a S P Y!
+Just like an S-P-Y!
```

Comment:
On the fence about a/an here. It's obviously `a spy`, but `an ess pee why`. I might lean more towards `a`?

Reply:
she spells it out explicitly, we don't say "a IBM business", we say "an IBM business"